spec/services/polymorphic_manifest_builder_spec.rb

```diff
@@ -280,7 +280,7 @@ def build_file_set(id)
       expect(result['@id']).to eq "http://plum.com/concern/scanned_resources/1/manifest"
     end
     it "has a description" do
-      expect(result.description).to eq [record.description]
+      expect(result.description).to eq [record.description.first]
```

Comment:
They aren't equivalent. `result.description` is and Array. `record.description` is and RDF something something class. The test fails unless you cast that into an Array.